screens/Scan/ScanScreenController.ts

```diff
@@ -38,19 +59,34 @@ export function useScanScreen() {
     locationError.message = t('errors.locationDenied.message');
     locationError.button = t('errors.locationDenied.button');
   }
+  if (isNearByDevicesPermissionDenied) {
+    nearByDevicesPermissionError.message = t(
+      'errors.nearbyDevicesPermissionDenied.message'
+    );
+    nearByDevicesPermissionError.button = t(
+      'errors.nearbyDevicesPermissionDenied.button'
+    );
+  }
 
   return {
     locationError,
-
+    nearByDevicesPermissionError,
```

Comment:
where is this boolean used? I don't see any usage in this changes

Reply:
Good point, I see NearByPrompt component is being used to show the error. I'll remove it.